pkg/importer/dataStream.go

```diff
@@ -168,9 +192,14 @@ func (d dataStream) http() (io.ReadCloser, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "could not create HTTP request")
 	}
-	if len(d.accessKeyId) > 0 && len(d.secretKey) > 0 {
+
+	if header != nil {
+		// ToDo: add validations
```

Comment:
+1

Reply:
Changed to merging the headers, instead of overriding it.